solutions/1500-1599/1514-path-with-maximum-probability.md

```diff
@@ -111,8 +126,9 @@ double maxProbability(int n, vector<vector<int>>& edges, vector<double>& succPro
 
 </Tabs>
 
-## Time complexity:  
-O(e*log(n)) where n -> nodes, e->edges  
+## Time complexity:    
+
+O(total_edges * log(n))  
   
 ## Space complexity:
-O(n) where n->nodes
+O(n)
```

Comment:
1. Apply LaTex ... 
2. state what `n` and `e` are

e.g.

```
## Time Complexity

$O(e*log(n))$ where $e$ is xxxx and $n$ is xxxxx
```

which will be rendered as

## Time Complexity

$O(e*log(n))$ where $e$ is xxxx and $n$ is xxxxx